integration-cli/docker_cli_cp_test.go

```diff
@@ -472,7 +472,7 @@ func TestCpVolumePath(t *testing.T) {
 	}
 
 	cleanedContainerID := strings.TrimSpace(out)
-	defer deleteContainer(cleanedContainerID)
+	defer dockerCmd(t, "rm", "-fv", cleanedContainerID)
```

Comment:
its gotta be something with creating the volume via vfs in overlay in overlay that makes the container need a SIGTERM and SIGKILL